tracker.go

```diff
@@ -85,6 +86,7 @@ func NewTracker(cap int) (*Tracker, error) {
 		pos:    0,
 		ch:     make(chan *request, cap),
 		done:   make(chan struct{}),
+		resp:   &PlainResponder{},
```

Comment:
can we make the responder part of the NewTracker? and not have the set method?

otherwise one can change state on an already instantiated tracker

Reply:
Sure! Your lib, your call. :)

PTAL